modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1598,10 +1598,6 @@ def shuffle_partitions(
             ]
         ).T
         # We need to convert every partition that came from the splits into a full-axis column partition.
-        col_partitioning_func = np.vectorize(
-            lambda partition: cls._row_partition_class(partition)
-        )
-        split_row_partitions = col_partitioning_func(split_row_partitions)
```

Comment:
Each partition from `split_row_partitions` is already a row partition at this point since we did the conversion before: 
https://github.com/modin-project/modin/blob/cdedd710f2ea092f72316600335b9e8e15995d28/modin/core/dataframe/pandas/partitioning/partition_manager.py#L1584-L1590
There's no need to do double-wrapping.

Reply:
Makes sense!